product_pricelist_extended/models/product_pricelist.py

```diff
@@ -11,7 +11,6 @@
 from odoo.exceptions import UserError, ValidationError
 
 # noinspection PyUnresolvedReferences
```

Comment:
If you are deleting the line the inspection override applies to, delete the inspection too.